config/config.go

```diff
@@ -776,6 +797,12 @@ func (c *Config) ToOptions() skipper.Options {
 		}
 	}
 
+	if c.ForwardedHeaders != (net.ForwardedHeaders{}) {
+		options.CustomHttpHandlerWrap = func(handler http.Handler) http.Handler {
```

Comment:
I wonder if we should allow a list here in order to allow custom skipper's to add a custom middleware chain here.

Reply:
We create options in this method
https://github.com/zalando/skipper/blob/1dc62b86f1d9d406de63f7fea392e6f0b8c24448/config/config.go#L559

It is the potential user that call `ToOptions()` should wrap if they want to keep the wrap intact

Reply:
@AlexanderYastrebov yes they can, but in case `c.ForwardedHeaders != (net.ForwardedHeaders{})`, we overwrite it or our ForwardedHeaders handler does not work as configured.

Reply:
We overwrite default value because we create `skipper.Options` anew

Reply:
thanks, true